Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -176,6 +176,19 @@ public virtual void OnValidate()
             }
         }
 
+#if UNITY_EDITOR
+        void Reset()
```

Comment:
it's not obvious why Reset() is used over OnValidate.
shouldn't it be OnValidate, and Reset() calls it too?
or at least a comment to explain why Reset?

Reply:
This was answered in the original PR...Reset fires first, OnValidate is too late to block the addition of the component and prevent the addition of the transport that we add if none is present.